crates/rpc/src/engine/mod.rs

```diff
@@ -1 +1,41 @@
-pub(crate) mod capabilities;
+use serde_json::{json, Value};
+use tracing::info;
+
+use crate::RpcErr;
+
+type ExchangeCapabilitiesRequest = Vec<String>;
+
+pub fn exchange_capabilities(params: Option<Value>) -> Result<Value, RpcErr> {
+    if let Some(params) = params {
+        let params: ExchangeCapabilitiesRequest = serde_json::from_value(params).unwrap();
```

Comment:
We shouldn't `unwrap` here. We can use `Option::map` and `Option::unwrap_or` to avoid it.